README.md

```diff
@@ -4,10 +4,9 @@
   </a>
 </p>
 
-**Celo Monorepo - Official repository for core projects comprising the Celo platform**
+**Valora Monorepo - Official repository for Valora**
 
-This repository contains the source code for the Celo core projects including the [smart contracts](https://github.com/celo-org/celo-monorepo/tree/master/packages/protocol), [wallet app](https://github.com/celo-org/celo-monorepo/tree/master/packages/mobile),
-and other packages. The source code for the Celo Blockchain which operates a node on the Celo Network is kept in a separate repo [here](https://github.com/celo-org/celo-blockchain).
+This repository contains the source code for Valora, on both iOS and Android. 
```

Comment:
Consider putting a sentence or two here about what Valora is.
Something like this: https://github.com/celo-org/celo-monorepo/pull/7600/files#diff-45798d5ec1a1e504b1d5d8ed55ba1e4353449e61b47eeffe1de4e727d1c96bcfR15

Reply:
sounds good!